upload_multiple_files_to_single_record.py

```diff
@@ -0,0 +1,224 @@
+import requests
+import hashlib
+from pathlib import Path
+from defusedxml.lxml import tostring
+from lxml.builder import E
+import mimetypes
+
+
+SITE_URL = 'https://library.tind.io'
+API_KEY = ''
+CALLBACK_EMAIL = 'demo@tind.io'
+OBJECT_STORE_NAME = 'TOS'
+
+APPEND_METADATA = {"245__a": "Test record",
+                   "269__a": "2021-10-14",
+                   "980__a": "DIGITIZED"}
+
+
+def request_presigned_object():
+    """Returns a presigned object"""
+    url = requests.compat.urljoin(SITE_URL, 'storage/presigned_post')
+    params = requests.compat.urlencode({'location': OBJECT_STORE_NAME})
+    url = "{}?{}".format(url, params)
+
+    headers = {'Authorization': 'Token ' + API_KEY}
+
+    r = requests.post(url, headers=headers)
+
+    response_object = r.json()
+    return response_object
+
+
+def upload_single_file(file_name, presigned_response):
+    """Post a file to S3 and returns the upload response"""
+    url = presigned_response["data"]["url"]
+    headers = {'x-amz-acl': 'private'}
+    data = presigned_response["data"]["fields"]
+    files = {'file': open(file_name, 'rb')}
+    # Time out if the request has not been able to communicate with the server in 10 seconds
+    upload_response = requests.post(url, headers=headers, files=files, data=data, timeout=10)
+
+    print(upload_response.status_code)
+    return upload_response
+
+
+def get_md5_checksum(file_path):
+    """
+    Get the md5 checksum of the local file.
+    """
+    return hashlib.md5(open(file_path, 'rb').read()).hexdigest()
+
+
+
+def compare_checksum(local_checksum, upload_response):
+    """
+    Comparing the checksum of the local file with the returned checksum/Etag
+    stored in the upload response.
+    """
+
+    etag = upload_response.headers.get('Etag', '')
+    upload_checksum = etag.replace('"', '')
+
+    if local_checksum == upload_checksum:
+        return True
+    return False
+
+
+def create_subfield(sub_code, content):
+    """
+    Create an empty subfield with a subfield code
+    """
+    new_subfield = E.subfield(code='{}'.format(sub_code))
+    new_subfield.text = content
+    return new_subfield
+
+
+def create_datafield(marc_key, subfield_tuple_lists=False):
+    """
+    marc_key: Five letter value: E.g. 245__
+    The subfield_tuple_list is on the form:
+    [('a', 'subfield text A'),
+     ('b', 'subfield text B')]
+    """
```

Comment:
If you use the default value of `subfield_tuple_lists` here, you'll get a `TypeError: 'bool' object is not iterable`. I suggest to do:

```suggestion
def create_datafield(marc_key, subfield_tuple_lists=None):
    """
    marc_key: Five letter value: E.g. 245__
    The subfield_tuple_list is on the form:
    [('a', 'subfield text A'),
     ('b', 'subfield text B')]
    """
    if subfield_tuple_lists is None:
        subfield_tuple_lists = []
```